### DISCUSSION TITLE
Icon styles sometimes break down

### DISCUSSION THREAD

**Discussion by plyaskin** [bagisto/bagisto#9920] [General]
### Terms

- [X] Before you openning this issue, i have checked if the issue has already been reported.

### Bagisto Version(s) affected

2.1.2

### Issue Description

Sometimes icons break down (Admin panel and Shop)

### Preconditions

Nothing

### Expected Result

No broken

### Actual Result

Broken

### Screenshots

![image](https://github.com/bagisto/bagisto/assets/9464550/499834a7-fe3e-4c4d-865d-2057b19298b9)

### Additional context

It solving after reloading the page.

**Comment by ashishkumar-webkul** [comment]
@plyaskin 

As per checking the concern related to the icons breakdown. we will discuss the same with team and we will get back to you for the same.
Thank you.

**Comment by devansh-webkul** [comment]
@plyaskin,

We are aware of this, but we are trying to reproduce the case and haven't been able to pinpoint the issue. It would be great if you could help us here. We think this might be a network issue, such as fonts not downloading properly.

**Comment by devansh-webkul** [comment]
This might be a issue due to failing request, i guess mostly this will happen in local not in live.
